host-bmc/utils.cpp

```diff
@@ -92,11 +107,24 @@ void addObjectPathEntityAssociations(
             {
                 continue;
             }
+            /*If chassis number is not empty, it means we've encountered
+              numbered chassis. So we prepend previous entity instance id and
+              chassis number to prepare unique leafnodes
+            */
```

Comment:
please use consistent commenting style throughout the entire file. I see that the style was being changed down below.